project_setup.md

```diff
@@ -43,21 +30,15 @@ help you decide which tool to use for packaging.
 - The testing framework used is [PyTest](https://pytest.org)
   - [PyTest introduction](https://pythontest.com/pytest-book/)
   - PyTest is listed as a development dependency
-  - This is configured in `setup.cfg`
+  - This is configured in `pyproject.toml`
```

Comment:
This `project_setup.md` file is new to me. What does it add to the README?

Reply:
Good question. This file is taken from the python template of eScience center, which helps on setting up the repo. I don't think it is very necessary if developers are already familiar with github and python repos, but it's nice for new developers.